robot/markers.py

```diff
@@ -1,30 +1,16 @@
 import math
+from collections import namedtuple
+from typing import List, NewType, Tuple
 
 from robot.game_specific import TOKEN, WALL
 
+TMetres = NewType('TMetres', float)
+TDegrees = NewType('TDegrees', float)
+TRadians = NewType('TRadians', float)
+TPixel = NewType('TPixel', float)
```

Comment:
nitpick: any reason that `Pixel` is singular, while the other unit-types are plural?